tests/test_routes.py

```diff
@@ -0,0 +1,53 @@
+def test_get_one_planet(client, two_saved_planets):
+    # Act
+    response = client.get("/planets/1")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 200
+    assert response_body == {
+        "id": 1,
+        "name": "Sun",
+        "description":"Fire and warmth 4evr",
+        "color": "Orange"
+    }
+
+def test_get_all_planets_with_no_records(client):
+    # Act
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 200
+    assert response_body == []
+
+def test_get_all_planets_with_records(client, two_saved_planets):
+    # Act
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 200
+    assert response_body == [
+        {
+        "id": 1,
+        "name": "Sun",
+        "description":"Fire and warmth 4evr",
+        "color": "Orange"
+        },
+        {
+        "id": 2,
+        "name": "Venus",
+        "description":"Ruler of Libra and Taurus",
+        "color": "Purple"
+        }
+    ]
+
+def test_get_one_planet_with_no_data(client):
```

Comment:
👍 Nice work getting tests up.  Future tests would include at least 1 for each of the 400 errors.